about.html

```diff
@@ -29,37 +29,20 @@
                     <li><a href="">sponsor</a></li
                     <li><a href="#">News</a></li>
                 </ul>
-                <li><a href=""> <img class="logo" src="./images/logo.png" alt=""></a> </li>  
             </div>
       
     </header>
-    <section class="container contain">
-        <div class="mobi-logo">
-            <img class="logo-1" src="./images/mobile_menu 1.png" alt="">
-           </div>
-          <div class="card-1">
-            <div class="introductory"> 
-              <h1 class="heading-1">"Hetroxity Academy"</h1>
-                  <h2 class="heading-2">Discover various computer 
-                  programming <br> courses and all in between</h2>
-            <div class="article-1">
-                 <p class="intro">
-                  want to land you first job ?  or Are you looking forward <br>
-                  to upgrade you career. Get Access to various courses <br>
-                   that will fulfill your  dream 
-                </p>
-            </div>
-            <div class="next">
-               <h3 class="h-next">Automation with python</h3>
-               <p class="h-next"> starts August 15th 2023</p>
-               <p class="h-next-1">Enroll now</p>
-            </div>
-            </div>
-        </div>
-    </section>
 
+        <div class="ham">
+            <img src="./images/mobile_menu 1.png" alt="">
+        </div>
+       
+<div class="frame-1">
     <section class="about-page" id="about-page">
         <h1 class="heading-1">"Hetroxity Academy"</h1>
+        <div class="yy">
+            life changing platform . discover your full potential
+        </div>
        <div class="abt-p">
```

Comment:
- [ OPTIONAL] Previous reviewer did not mention but this part is of about me is totally different from the one in the design. Please fix it as well so that the image and footer / partner's are both position as well as the provided design. 

![image](https://github.com/hetrox8/First-Capstone-Project/assets/43877689/e34e7cba-f8d8-4e06-8241-dda93dd568cf)